### ISSUE TITLE
AudioContext#resume should be moved to OfflineAudioContext#resume

### ISSUE THREAD

**Issue by saschanaz** [mdn/browser-compat-data#6733]
@foolip, mdn-bcd-collector currently says both `AudioContext.resume` and `OfflineAudioContext.resume` are available on Firefox but actually they are from `BaseAudioContext`. Should it check their existence by `hasOwnProperty`?

(Chrome somehow does have separate `AudioContext.resume` and `OfflineAudioContext.resume` 🤔)

**Comment by foolip** [comment]
@saschanaz mdn-bcd-collector is intentionally using more lax tests like `'foo' in Bar.prototype`, closer to the feature detection a web developer might use. In fact, where possible we'd like to test instances and not prototypes, and the current test for `AudioContext.resume` is:
```js
(function () {
var instance = new (window.AudioContext || window.webkitAudioContext)();
return 'resume' in instance;
})();
```

The reason for this is that there are so many little mishaps in implementations and differences in which prototype things live on that ultimately don't matter to the question "can I use this?"

However, *sometimes* a situation like this warrants a note. Currently we don't discover the situation automatically, but we could: https://github.com/foolip/mdn-bcd-collector/issues/523

**Comment by foolip** [comment]
As for AudioContext#resume and OfflineAudioContext#resume, both exist in the spec:
https://webaudio.github.io/web-audio-api/#AudioContext
https://webaudio.github.io/web-audio-api/#OfflineAudioContext

If Firefox has put these on `BaseAudioContext`, I don't think that really matters to web developers, right?

**Comment by saschanaz** [comment]
Ah cool then, let's mark the OfflineAudioContext one as supported in Firefox and I'll be good then.

**Comment by foolip** [comment]
@vinyldarkscratch https://mdn-bcd-collector.appspot.com/tests/api/OfflineAudioContext says this is supported in Firefox, and I made an attempt to quickfix, but ran into https://github.com/foolip/mdn-bcd-collector/issues/531. This is probably not an isolated problem, so would you be able to self-assign both this and that issue, and use this as the example to prove it working better?

**Comment by foolip** [comment]
With the fixes in https://github.com/foolip/mdn-bcd-collector/pull/532 the update script does set OfflineAudioContext#resume to added in Firefox 40. @vinyldarkscratch I won't send a PR for just one entry, can you bake that into a larger PR fixing things with Web Audio?

**Comment by Elchi3** [comment]
This looks fixed to me. Please reopen if not.

**Comment by foolip** [comment]
Yep, the data was updated in https://github.com/mdn/browser-compat-data/pull/14291.
